ddmrp/models/stock_warehouse_orderpoint.py

```diff
@@ -619,6 +645,7 @@ def cron_ddmrp(self, automatic=False):
         orderpoints = self.search([])
         i = 0
         j = len(orderpoints)
+        orderpoints.refresh()
```

Comment:
Having the refresh here instead of in each `_calc` method reduces the `test_procure_recommended` from ~8s to ~4s on my machine. Tests still pass but I'm not 100% confident there is no side effect.

Reply:
I think there is no side effect, good for me